code/modules/projectiles/bullet.dm

```diff
@@ -50,9 +50,10 @@ toxic - poisons
 	hit_mob_sound = 'sound/impact_sounds/Flesh_Stab_2.ogg'
 
 //Any special things when it hits shit?
-	on_hit(atom/hit, direction, projectile)
+	on_hit(atom/hit, direction, obj/projectile/P)
 		if (ishuman(hit) && src.hit_type)
 			take_bleeding_damage(hit, null, round(src.power / 3), src.hit_type) // oh god no why was the first var set to src what was I thinking
+			hit.changeStatus("staggered", clamp(P.power, 5, 1) SECONDS)
```

Comment:
Looking at this, it seems like almost every hit should apply a 5 second stagger since almost every projectile in the game has a power value > 5. Even the noble staple "bullet", which is arguably the weakest projectile in the game. Same applies to energy bolt projectiles, I think tasers have a power of 20. I think there should be a bit more variety in terms of projectiles applying the stagger effect on hit.

Reply:
Whoops, that was supposed to be `power/8`

Reply:
Also, this applys only to 'bullet' projectiles (not taser/wavegun/laser/etc)